packages/cf-builder-card/src/cardReducer.js

```diff
@@ -1,10 +1,12 @@
-const ActionTypes = require('./CardActionTypes');
+// @flow
 
-const initialState = {};
+import type {CardAction, CardState} from './CardTypes';
```

Comment:
ES6 modules are standardized, how to load them is not since there are 3 (4?) related proposals floating around. But it doesn't matter since we don't use any of the proposed loading mechanism.